app/views/layouts/offices.html.erb

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>MrClean</title>
+    <%= csrf_meta_tags %>
+
+    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+  </head>
+
+  <body>
+    
+    
+    <% if user_signed_in? %>
+      <%= link_to "Log Out", destroy_user_session_path %>
+    <% else %>
+      <%= link_to "Log In", new_user_session_path %>
+    <% end %>
+
+    <p class="notice"><%= notice %></p>
+    <p class="alert"><%= alert %></p>
+
+
+    <%= yield %>
+
+  </body>
+</html>
```

Comment:
I prefer that we use the application layout and modify it as needed. Also, iIs this used anywhere?